src-tauri/src/auth.rs

```diff
@@ -59,6 +61,28 @@ pub(crate) async fn refresh_token(refresh_token: String) -> AuthenticationRespon
 	create_token(client, body).await
 }
 
+pub(crate) async fn get_access_token_for_papi() -> String {
+	let client = Client::new();
+	let body = format!(
+		"client_id={}&client_secret={}&grant_type=client_credentials",
+		ENVIRONMENT_VARIABLES.oidc_tekst_client_id, ENVIRONMENT_VARIABLES.oidc_tekst_client_secret
+	);
+
+	let res = client
+		.post(format!(
+			"{}{}",
+			ENVIRONMENT_VARIABLES.oidc_tekst_base_url, "/token"
+		))
+		.header("Content-Type", "application/x-www-form-urlencoded")
+		.body(body)
+		.send()
+		.await;
+
+	let token_response: TokenResponseWithoutRefresh =
+		serde_json::from_str(&res.unwrap().text().await.unwrap()).unwrap();
```

Comment:
Ser dette problemet gjelder flere plass i denne filen. Men alle disse unwrappene har vel potensiale til å gi en error ut, altså gi programmet "panic" her. Så det må nok feilhåndteres bedre. Trenger ikke at Trøkk crasher hvis dette feiler.

Så enten bruke "match" for å se om det feiler (Err) og håndtere. Eller la denne funksjonen returnere ett Result og videresende feilen og håndter der den blir brukt.

Reply:
Tar bare denne feilen i denne saken, lager en egen sak for å fikse unwrap feil (TT-1433). Se ny commit.